.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: c
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
+script: bash -ex .travis-opam.sh
+sudo: required
+env:
+  - PACKAGE="tlstunnel" OCAML_VERSION=4.01   POST_INSTALL_HOOK="./.travis-test.sh"
```

Comment:
i think this one will be travis-only, since it'll eventually generate cert, start a daemon, etc.

> On 16 Apr 2015, at 09:26, Thomas Gazagnaire <notifications@github.com> wrote:
> 
> In .travis.yml <https://github.com/hannesm/tlstunnel/pull/12#discussion_r28491735>:
> 
> > @@ -0,0 +1,7 @@
> > +language: c
> > +install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
> > +script: bash -ex .travis-opam.sh
> > +sudo: required
> > +env:
> > +  - PACKAGE="tlstunnel" OCAML_VERSION=4.01   POST_INSTALL_HOOK="./.travis-test.sh"
> Would be better to move the POST_INSTALL_HOOK command into the build-test field of the opam file.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/hannesm/tlstunnel/pull/12/files#r28491735>.
>